reflex/components/graphing/recharts/cartesian.py

```diff
@@ -195,6 +196,10 @@ class Cartesian(Recharts):
     # The type of icon in legend. If set to 'none', no legend item will be rendered. 'line' | 'plainline' | 'square' | 'rect'| 'circle' | 'cross' | 'diamond' | 'star' | 'triangle' | 'wye' | 'none'optional
     # legend_type: Var[LiteralLegendType]
 
+    def _apply_theme(self, theme: Component):
+        self.stroke = Var.create(f"var(--accent-8)")
```

Comment:
This doesn't seem to be working... when i change the accent color, it does not affect the charts